Adafruit_VEML7700.h

```diff
@@ -101,7 +104,7 @@ class Adafruit_VEML7700 {
     *ALS_Persistence, *ALS_Integration_Time, *ALS_Gain,
     *PowerSave_Enable, *PowerSave_Mode;
 
-  float normalize(float value);
+  float normalize_resolution(float value);
```

Comment:
why rename this? keep previous name, its private

Reply:
I just thought it would be confusing to have two new normalization functions alongside a 'normalize' function that they don't interact with 🤷‍♂

Reply:
oki i guess it isnt exposed, it doesnt matter